ecc/ecc-key-decode.c

```diff
@@ -0,0 +1,84 @@
+#include <wolfssl/options.h>
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfssl/wolfcrypt/ecc.h>
+#include <wolfssl/wolfcrypt/asn.h>
+#include <wolfssl/wolfcrypt/logging.h>
+#include <wolfssl/wolfcrypt/wc_port.h>
+
+#include <stdio.h>
+
+#if defined(WOLFSSL_TEST_CERT) || defined(OPENSSL_EXTRA) || \
+    defined(OPENSSL_EXTRA_X509_SMALL)
+
+static void check_ret(char*, int);
+
+int main(void)
+{
+    DecodedCert decodedCert;
+    FILE* file;
+    byte derBuffer[4096];
+    size_t bytes;
+    ecc_key eccKey;
+    int ret;
+    word32 inOutIdx = 0;
+
+#ifdef DEBUG_WOLFSSL
+    wolfSSL_Debugging_ON();
+#endif
+
+    wolfCrypt_Init();
+
+    file = fopen("../certs/client-ecc-cert.der", "rb");
+    if (!file) {
+        printf("Failed to open file\n");
+        exit(-99);
+    }
+
+    bytes = fread(derBuffer, 1, 4096, file);
+    fclose(file);
+
+    printf("bytes = %d\n", (int) bytes);
```

Comment:
Good human-check can confirm if bytes read was zero, for completeness could add an error check